### ISSUE TITLE
Upgrade Docker image to Java 11

### ISSUE THREAD

**Issue by gaul** [gaul/s3proxy#397]
Can someone from the community test this?

**Comment by timuralp** [comment]
There is no `opendjk:11-jre-alpine image`, so I tested `openjdk:11-jre-slim`, which appears to work as expected (only did a small test that the container starts and I can create bucket and PUT an object). This image is a little bigger -- 243MB. There is an alpine image from the eclipse temurin project (`eclipse-temurin/openjdk:11-jre-alpine`). This similarly appears to work and results in a 149MB image.

**Comment by gaul** [comment]
I think that getting on a more recent Java version is more important than minimizing binary size.  The true solution to this is [jlink](https://docs.oracle.com/javase/9/tools/jlink.htm) to avoid bundling the entire JDK but I don't understand how to do this.

During earlier s3fs integration tests I found that S3Proxy using Java 11 performed somewhat faster than 8 (maybe better hashing vectorization?).  I did not test 17 since it requires a new jclouds release or local dependency override #396.
